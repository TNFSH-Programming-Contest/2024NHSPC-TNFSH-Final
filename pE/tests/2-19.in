1 2878493 4478609 7434580 8233922
92
