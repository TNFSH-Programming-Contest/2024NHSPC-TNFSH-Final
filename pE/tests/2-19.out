8433051396512820
